src/main/kotlin/kotlinx/kover/appliers/KoverMergedApplier.kt

```diff
@@ -92,6 +92,8 @@ private class ProcessMergeExtensionAction(private val extension: KoverMergedConf
         // TODO `onlyIf` block moved out from config lambda because of bug in Kotlin compiler - it implicitly adds closure on `Project` inside onlyIf's lambda
         verifyTask.onlyIf { extension.verify.hasActiveRules() }
 
+        // ordering of task calls, so that if a verification error occurs, reports are generated and the values can be viewed in them
+        verifyTask.shouldRunAfter(xmlTask, htmlTask)
```

Comment:
Does it fix the situation when a unit test in `check` fails and a report is not generated?

If so, a test for that would be nice

Reply:
Yes, #209  is just about that

Reply:
Unfortunately, if the test has fallen, then no, this is a Gradle limitation.
The report generation task depends on the test task, if any of the tasks failed, then Gradle stops executing all dependent tasks

Reply:
Is it true for `--continue` as well?

Reply:
According [docs](https://docs.gradle.org/current/userguide/command_line_interface.html#sec:continue_build_on_failure) `If a task fails, any subsequent tasks that were depending on it will not be executed.` - because when generating reports, the output of test tasks is used, so even in this case they will not be executed

Reply:
Can you still add a test on the change though?

Reply:
Ready